forms/serializers/form.py

```diff
@@ -750,6 +750,25 @@ def update(self, instance, validated_data):
         return instance
 
 
+class AnswerPublicSerializer(AnswerSerializer):
```

Comment:
this raise NotImplementedError is very ugly way to do this. I would just restrict this via view.

Reply:
Lets do so, sounds good

Reply:
@vinikvuo I provided a fix, could you check that